pkgs/games/openmw/default.nix

```diff
@@ -13,56 +13,75 @@
 , unshield
 , openal
 , libXt
+, lz4
+, recastnavigation
 }:
 
 let
-  openscenegraph_ = openscenegraph.overrideDerivation (self: {
+  openscenegraph_openmw = (openscenegraph.override { colladaSupport = true; })
+    .overrideDerivation (self: {
+      src = fetchFromGitHub {
+        owner = "OpenMW";
+        repo = "osg";
+        rev = "bbe61c3bc510a4f5bb4aea21cce506519c2d24e6";
+        sha256 = "sha256-t3smLqstp7wWfi9HXJoBCek+3acqt/ySBYF8RJOG6Mo=";
+      };
+    });
+
+  bullet_openmw = bullet.overrideDerivation (old: rec {
+    version = "3.17";
```

Comment:
Can we update bullet or add buller_3 instead?

Reply:
I don't know. I'll probably take a look at that later, but seeing there are other software that depend on bullet, and that it seems to be a major new version, I plan to make another PR dedicated to this.

Reply:
Well, it seems updating the library doesn't cause any build failure nor cause runtime failure on vdrift or stuntrally. I'll open a different PR for it,

Reply:
Done here : https://github.com/NixOS/nixpkgs/pull/145385